src/video/cocoa/cocoa_v.mm

```diff
@@ -135,6 +138,30 @@
 	this->UpdateAutoResolution();
 	this->orig_res = _cur_resolution;
 
+	/* Set up gamepad connection/disconnection notifications */
+	[[NSNotificationCenter defaultCenter] addObserverForName:GCControllerDidConnectNotification
```

Comment:
Just as an observation, for hysterical raisins, we have a bit of an abnormal code formatting style for Obj-C, especially in the area of [ ].

Not sure though if it is worth keeping it to be consistent or if we should just ditch it long-term.